packages/image_picker/image_picker/ios/Classes/FLTPHPickerSaveImageToPathOperation.m

```diff
@@ -84,16 +84,28 @@ - (void)start {
   }
   if (@available(iOS 14, *)) {
     [self setExecuting:YES];
-
-    if ([self.result.itemProvider hasItemConformingToTypeIdentifier:UTTypeWebP.identifier]) {
-      [self.result.itemProvider
-          loadDataRepresentationForTypeIdentifier:UTTypeWebP.identifier
-                                completionHandler:^(NSData *_Nullable data,
-                                                    NSError *_Nullable error) {
-                                  UIImage *image = [[UIImage alloc] initWithData:data];
-                                  [self processImage:image];
-                                }];
-      return;
+    NSArray *supportedRepresentations = @[
+      UTTypeRAWImage.identifier,
+      UTTypeTIFF.identifier,
+      UTTypeBMP.identifier,
+      UTTypePNG.identifier,
+      UTTypeHEIF.identifier,
+      UTTypeHEIC.identifier,
+      UTTypeJPEG.identifier,
+      UTTypeWebP.identifier,
+      UTTypeGIF.identifier,
```

Comment:
Why are we changing the codepath for types that already worked? Is this a preferred API?